simulators/portal-interop/src/main.rs

```diff
@@ -480,3 +492,59 @@ dyn_async! {
         }
     }
 }
+
+dyn_async! {
+    // test that a node will return a receipts via FINDCONTENT over uTP that it has stored locally
+    async fn test_find_content_receipts_over_utp<'a> (client_a: Client, client_b: Client) {
+        let receipts_key: HistoryContentKey = serde_json::from_value(json!(RECEIPTS_KEY)).unwrap();
+        let receipts_value: HistoryContentValue = serde_json::from_value(json!(RECEIPTS_VALUE)).unwrap();
+
+        let header_with_proof_key: HistoryContentKey = serde_json::from_value(json!(HEADER_WITH_PROOF_KEY)).unwrap();
+        let header_with_proof_value: HistoryContentValue = serde_json::from_value(json!(HEADER_WITH_PROOF_VALUE)).unwrap();
+
+        match client_b.rpc.store(receipts_key.clone(), receipts_value.clone()).await {
+            Ok(result) => if !result {
+                panic!("Error storing header with proof for Receipts verification");
```

Comment:
Error storing receipts rather?

Reply:
My bad xD I didn't notice that, resolved